test/jit/test_builtins.py

```diff
@@ -74,6 +74,39 @@ def forward(self, name):
         with self.assertRaisesRegex(RuntimeError, "hasattr"):
             torch.jit.script(Mod())
 
+    def test_del_single_operand(self):
```

Comment:
Leave the single operand tests in `test_list_dict.py`; only put the multiple operand tests here

Reply:
Done.